CefSharp.BrowserSubprocess.Core/BindObjectAsyncHandler.h

```diff
@@ -228,23 +191,13 @@ namespace CefSharp
                             else
                             {
                                 //Objects already bound or ignore cache
-                                CefRefPtr<CefV8Value> promiseResolve;
-                                CefRefPtr<CefV8Exception> promiseException;
+                                auto promiseResolve = CefV8Value::CreatePromise();
```

Comment:
You create a promise and assign it to the result three times. You can create it earlier and assign it to `retval`, and resolve it later. It will work.
And we can replace `exception = "BindObjectAsyncHandler::Execute - Browser wrapper null, unable to bind objects";` with `promiseResolve->RejectPromise("BindObjectAsyncHandler::Execute - Browser wrapper null, unable to bind objects");`.